scripts/e2e.js

```diff
@@ -0,0 +1,187 @@
+const { Api } = require('@cennznet/api');
+const { Keyring } = require('@polkadot/keyring');
+
+async function main() {
+
+    const [deployer] = await ethers.getSigners();
+    console.log('Deployer::',deployer.address);
+    const Bridge = await ethers.getContractFactory('CENNZnetBridge');
+    console.log('Deploying CENNZnet bridge contract...');
+    const bridge = await Bridge.deploy();
+    await bridge.deployed();
+    console.log('CENNZnet bridge deployed to:', bridge.address);
+
+    const erc20Peg = await ethers.getContractFactory('ERC20Peg');
+    console.log('Deploying ERC20Peg contract...');
+    const peg = await erc20Peg.deploy(bridge.address);
+    await peg.deployed();
+    console.log('CENNZnet erc20peg deployed to:', peg.address);
+
+    const TestToken = await ethers.getContractFactory('TestToken');
+    console.log('Deploying TestToken contract...');
+    const token = await TestToken.deploy("1000000");
+    await token.deployed();
+    console.log('TestToken deployed to:', token.address);
+
+    console.log(await peg.activateDeposits());
+
+    /***************** Make Deposit on ETHEREUM **********************/
+
+    let depositAmount = 5644;
+    // Alice
+    let cennznetAddress = "0xd43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d";
+    console.log(await token.approve(peg.address, depositAmount));
+    const deposition = await peg.deposit(token.address, depositAmount, cennznetAddress);
+    const depositTxHash = deposition.hash;
+    console.log('depositTxHash:::',depositTxHash);
+
+
+
+    /***************** CENNZnet Setup **********************/
+        // connecting to local chain
+    const api = await Api.create();
+    const keyring = new Keyring({type: 'sr25519'});
+    const alice = keyring.addFromUri('//Alice');
+    const transaction1 = api.tx.erc20Peg.activateDeposits(true);
+    const transaction2 = api.tx.erc20Peg.activateWithdrawals(true);
+    const transaction3 = api.tx.erc20Peg.setContractAddress(peg.address);
+    const transaction4 = api.tx.ethBridge.setEventConfirmations(0); // Hardhat only makes blocks when txs are sent
+    const batchBridgeActivationEx = api.tx.utility.batch([
+        transaction1,
+        transaction2,
+        transaction3,
+        transaction4
+    ]);
```

Comment:
could you add a comment about this only being necessary for a local chain, where it's not configured